api/types/types.proto

```diff
@@ -2556,7 +2556,9 @@ message OIDCConnectorSpecV3 {
     // RedirectURL is a URL that will redirect the client's browser
     // back to the identity provider after successful authentication.
     // This should match the URL on the Provider's side.
-    string RedirectURL = 4 [ (gogoproto.jsontag) = "redirect_url" ];
+    //
+    // DELETE IN 11.0.0 in favor of RedirectURLs
+    string RedirectURL = 4 [ (gogoproto.jsontag) = "-" ];
```

Comment:
What is the result of `[ (gogoproto.jsontag) = "-" ];`?

Reply:
It will be skipped when marshalling/unmarshaling, so we won't get the following unwanted result:
```json
kind: oidc
spec: 
    RedirectURL: ""
    redirect_url:
        - proxy.example.com...
        - other.example.com...
```